Quests/QuestNPC.cs

```diff
@@ -56,20 +57,73 @@ public BaseQuest GetCurrentQuest(NPC npc, out bool isDoing)
 
         public void TakeQuest()
         {
+            // Set the delay between Quests based on the Quest
+            switch (availableQuest.QuestName)
+            {
+                case "Tutorial 2":
+                    questCheckTick = 1200;
+                    break;
+                default:
+                    questCheckTick = 600;
+                    break;
+            }
```

Comment:
If we want variable delay after quest completion, it would be better to do it as a virtual property on the quest itself, which is available here.

Also just in case you should null-check `availableQuest` here.